### PR TITLE
Move `validate` tests to `base`

### LINKED ISSUE #173: Return unit tests of `validate` package back under the `base` module
See `smoke-tests/validation-rules/src/test/.../validate`. These tests should be under the main code tests.

Otherwise API of the validators become fragile — there is no way of knowing it's used even in our tests.

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we move some of the tests for `io.spine.validate` from `smoke-tests` back to `base`.

The only 2 kinds of tests retained in `smoke-tests` are tests for external validation rules and for `ValidatingBuilder`s generation.

Fixes #173.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, @alexander-yevsyukov, PTAL.

File: base/src/test/proto/spine/test/validate/commands.proto [hunk 1]
```diff
@@ -19,14 +19,14 @@
...
-package spine.test.validate.msg;
+package spine.test.validate;
 
 import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
 option java_multiple_files = true;
 option java_outer_classname = "ValidationTestCommandsProto";
-option java_package = "io.spine.test.validate.msg";
+option java_package = "io.spine.test.validate.commands";
```
Comment (armiol) [inline base/src/test/proto/spine/test/validate/commands.proto]: I'd use `command` instead of `commands`.

**Comment by armiol** [comment]
@dmdashenkov Let's kill `@author` mentions. And see my only comment.

**Comment by dmdashenkov** [comment]
@armiol, PTAL again.

**Pull request merged**